data/yaml/name_variants.yaml

```diff
@@ -5511,6 +5511,9 @@
 - canonical: {first: Yang, last: Liu}
   comment: 3M Health Information Systems
   id: yang-liu-3m
+- canonical: {first: Yang, last: Liu}
+  comment: other
+  id: yang-liu-other
```

Comment:
For "unknown"s, I think we should just use [the base name](https://aclanthology.org/people/yang-liu), i.e., `id: yang-liu`.